roles/acme/tasks/preconditions.yml

```diff
@@ -27,3 +27,10 @@
     - challenge/dns-01/{{ acme_challenge_provider }}.yml
     - challenge/http-01/{{ acme_challenge_provider }}.yml
     - challenge-unknown.yml
+
+- name: Check if a acme_domain is set
```

Comment:
I think this should be move up to be task nr 2

Reply:
I changed this with https://github.com/telekom-mms/ansible-collection-acme/commit/541861ecf069d8da23ee3ca4f1d239ae296f3d8c